Cargo.toml

```diff
@@ -71,8 +72,10 @@ portaudio-backend = ["portaudio-rs"]
 pulseaudio-backend = ["libpulse-sys"]
 
 with-tremor = ["tremor"]
+with-rust-mdns = ["mdns"]
+with-avahi = ["dns-sd"]
```

Comment:
Same, `with-external-mdns`.

`dns-sd` is actually Apple's API.
Avahi provides a compatibility layer between the two